INSTALL.md

```diff
@@ -0,0 +1,60 @@
+# Installing Records Mover
+
+You can install records-mover with the following 'extras':
+
+* `pip3 install records-mover` - Install minimal version, not
+  including `pandas` (needed only for local data copy), `psycopg2`
+  (needed for Redshift or PostgreSQL connections) or `pyarrow` (needed
+  for local Parquet manipulation).
+* `pip3 install records-mover[gsheets]` - Minimal install plus API
+  libraries to access Google Sheets.
+* `pip3 install records-mover[mover-cli]` - Install everything and
```

Comment:
Is it mover-cli or movercli? I’ve seen it both ways in the docs (will flag)

Reply:
`movercli`.  Where else did you see `mover-cli` in the docs?